config-windows.md

```diff
@@ -82,6 +82,12 @@ The following parameters can be specified (mutually exclusive):
 * **`count`** *(uint64, OPTIONAL)* - specifies the number of CPUs available to the container. It represents the fraction of the configured processor `count` in a container in relation to the processors available in the host. The fraction ultimately determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles.
 * **`shares`** *(uint16, OPTIONAL)* - limits the share of processor time given to the container relative to other workloads on the processor. The processor `shares` (`weight` at the platform level) is a value between 0 and 10,000.
 * **`maximum`** *(uint16, OPTIONAL)* - determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles. Set processor `maximum` to a percentage times 100.
+* **`affinityCPUs`** *(array of objects, OPTIONAL)* - specifies the set of CPU to affinitize for this container.
+
+Each entry has the following structure:
+
+* **`CPUMask`** *(uint64, REQUIRED)* - specifies CPU mask relative to this CPU group.
+* **`CPUGroup`** *(uint32, REQUIRED)* - CPU group that this CPU belongs to.
```

Comment:
Someone coming in and reading this spec would have no idea what "mask" and "group" mean here. Can we define specifically what we're talking about, or link to a doc which does?

Basically think of the audience as someone who is familiar with containers, but not familiar with Windows CPU concepts. How do they know what these mean, and how they map to OS concepts?

Reply:
Updated with link to msdocs that explains these concepts as it might be too much to add into this file.